lib/matplotlib/colorbar.py

```diff
@@ -267,7 +267,8 @@ def __call__(self):
         vmin = self._colorbar.norm.vmin
         vmax = self._colorbar.norm.vmax
         ticks = ticker.AutoMinorLocator.__call__(self)
```

Comment:
[Algorithm Implementation - Wikibooks, open books for an open world.pdf](https://github.com/matplotlib/matplotlib/files/2386084/Algorithm.Implementation.-.Wikibooks.open.books.for.an.open.world.pdf)